src/hooks/useAddCasesInputs.tsx

```diff
@@ -84,6 +84,8 @@ const useAddCasesInputs = (
   // use the current state of the facility as default values
   const defaultInputs = facility.modelInputs;
 
+  let isReference = defaultInputs.isReference;
```

Comment:
you may have noticed the related lint warning already but I don't think this is going to do what you want as `observedAtVersion` changes because it doesn't have any persistent state across renders. 

Since it is more or less functionally equivalent to `observedAtVersion.isReference` I think you ought to just set it to that rather than have it be a separate stateful value?